### PR TITLE
Customize Language Selector viewlet to display it as a menu.

### PR DESCRIPTION
**Pull request by phgross**
The styling is implemented in the https://github.com/4teamwork/plonetheme.teamraum/pull/286 PR.

Additionally I've added french to the supported languages in the examplecontent profile  to make sure that the language selector is activated during development.

@deiferni please have a look ...

### CODE REVIEW

File: opengever/base/viewlets/language_selector.pt [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<div id="portal-languageselector-wrapper"
+     tal:define="languages view/languages;
+                 here_url context/absolute_url;
+                 current python:[lang for lang in languages if lang.get('selected')][0]"
```
Comment (deiferni) [inline opengever/base/viewlets/language_selector.pt]: This could be a viewlet method

Comment (phgross) [inline opengever/base/viewlets/language_selector.pt]: :+1:

**Comment by phgross** [comment]
@deiferni updated ...

**Comment by deiferni** [comment]
:+1:

**Pull request merged**